docs/dev/adding_instruments.rst

```diff
@@ -105,6 +105,13 @@ This is a minimal instrument definition:
 
 Make sure to include the PyMeasure license to each file, and add yourself as an author to the :code:`AUTHORS.txt` file.
 
+There is a certain order of elements in an instrument class that is useful to adhere to:
+
+* First, the initializer (the :code:`__init__()` method), this makes it faster to find when browsing the source code.
+* Then class attributes/variables, if you need them.
+* Then properties (pymeasure-specific or generic Python variants). This will be the bulk of the implementation.
+* Finally, any methods.
+
```

Comment:
Maybe this is also the place for hints about good naming patterns.
```suggestion

It's good to name properties self explaining, short, and descriptive. `voltage_setpoint`, for example, would be good for the output voltage of a voltage source, which can measure a voltage as well. Verbs are a bad choice, because a property is no action (even though it may require one to acquire the data), so the measured voltage property should be called `measured_voltage` or just `voltage`, but not `measure_voltage`.
Similarly, if you want the device to perform some action (starting/stopping something etc.), these methods (mostly without parameters and return values) should use verbs for their name, e.g. `start_scan()`.

```

Reply:
yes, as I said, I have some stuff locally, but it was too late yesterday. :-P
I'll set this PR to WIP or something :D

Reply:
They're just some suggestions, that you do not have to write everything yourself.

Reply:
Oh, great expansion. Thanks